README.md

```diff
@@ -132,6 +132,73 @@ For example:
 $ html-sketchapp --url http://localhost:3000 --out-dir dist --debug
 ```
 
+### Symbol Middleware
+
+Symbol Middleware allows you to call out to any APIs that may be exposed on the underlying html-sketchapp layer.
+
+The current usecase for this is the new `layer.setResizingConstraint` API which allows you to configure how a layer should behave when a symbol is resized.
+
+#### Requiring a file
+
+The below uses the string argument to `require` in a file that defines what resizing a layer should have applied to it. In the below case, fixing the layer to the top and left.
+
+```bash
+$ html-sketchapp --symbol-middleware symbol.middleware.js
+```
+
+```js
+module.exports = (args) => {
+  const { layer, RESIZING_CONSTRAINTS } = args;
+
+  layer.setResizingConstraint(RESIZING_CONSTRAINTS.LEFT, RESIZING_CONSTRAINTS.TOP);
+};
+```
+
+#### Inline function
+
+If you use the config file you can provide an inline function and avoid creating a separate file:
+
+```bash
+$ html-sketchapp --config config.js
+```
+
+```js
+module.exports = {
+  symbolMiddleware: (args) => {
+    const { layer, RESIZING_CONSTRAINTS } = args;
```

Comment:
I was going to suggest renaming `layer` to `symbol`, but after digging into the code, I realised that this operating on each layer within a symbol rather than the symbol itself. I wonder if we need to rename this to `symbolLayerMiddleware`, particularly as we may want to introduce middleware that operates on the final symbols in the future?

Reply:
yep that makes sense